tests/python/unittest/test_optimizer.py

```diff
@@ -835,8 +835,8 @@ def update(self, index, weight, grad, state):
         if self.clip_weights:
              mx.ndarray.clip(weight, -self.clip_weights, self.clip_weights, out=weight)
 
-@unittest.skip("Test fails intermittently. Temporarily disabled until fixed. Tracked at https://github.com/apache/incubator-mxnet/issues/8230")
-@with_seed(0)
+# @unittest.skip("Test fails intermittently. Temporarily disabled until fixed. Tracked at https://github.com/apache/incubator-mxnet/issues/8230")
```

Comment:
If you're getting rid of the line please delete it.

Reply:
+1

Reply:
done